packages/grid/backend/grid/start.sh

```diff
@@ -1,6 +1,7 @@
 #! /usr/bin/env bash
 set -e
 
+pip install nltk
```

Comment:
todo: this is not a great solution

Reply:
@khoaguin perhaps we can move this into data_science optional dependencies